app/views/user_mailer/password_changed.html.slim

```diff
@@ -3,10 +3,9 @@ html
     meta content='text/html; charset=UTF-8' http-equiv='Content-Type'
   body
 
-    p You have asked Upaya to change the password currently associated with your Upaya Account.
+    p You have asked #{APP_NAME} to change the password currently associated with your #{APP_NAME} Account.
 
     p
-      'If you did not change your password, please contact the Upaya Customer Contact Center via the web form at
-      = link_to 'https://upaya.18f.gov/contact', 'https://upaya.18f.gov/contact'
+      'If you did not change your password, please contact the #{APP_NAME} Customer Contact Center via the web form at = link_to 'https://upaya.18f.gov/contact', 'https://upaya.18f.gov/contact'
```

Comment:
Doesn't code climate to that for us?    I had been assuming that overcommit was mostly for lint the commit message